test_scripts/API/AutoCompleteList/009_ATF_SetGlobalProperties_P_AutoCompleteList_ArrayOfLowerBound.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
```

Comment:
@Aleksandriys, please update summary with one more requirement, related to CRQ:
[SetGlobalProperties] SDL must transfer request to HMI in case of valid "autoCompleteList" param